MatrixSDK/Utils/Media/MXMediaLoader.m

```diff
@@ -40,31 +40,22 @@
 
 NSString *const kMXMediaUploadIdPrefix = @"upload-";
 
-// Modified by Nedap. Store accessToken to set the authorization token on media requests (BER-229)
+
 @interface MXMediaLoader()
 
-@property (nonatomic) NSString *accessToken;
+@property (nonatomic, readonly) NSString* accessToken;
 
 @end
 
 @implementation MXMediaLoader
 
 @synthesize statisticsDict;
 
-- (id)init
+- (id)initWithAccessToken:(NSString *) accessToken
 {
     if (self = [super init])
     {
         _state = MXMediaLoaderStateIdle;
-    }
-    return self;
-}
-
-// Modified by Nedap. Init with access token to set the authorization token on media requests (BER-229)
-- (id)initWithAccessToken:(NSString *)accessToken
```

Comment:
please check this one as well

Reply:
Yes, I removed it because matrix also added `(id)initWithAccessToken:(NSString *) accessToken` so I kept their changes 😄